main.go

```diff
@@ -43,6 +44,6 @@ func main() {
 	userController.Route(app)
 
 	log.Fatal(
-		app.Listen(config.Get("SERVER_PORT")),
+		app.Listen(fmt.Sprintf(":%s", config.Get("SERVER_PORT"))),
```

Comment:
i think it's fine for now to use this, maybe on next feature we will replace `SERVER_POST` to `SERVER_HOST`